### PR TITLE
Update dartdevc issue label

### PR DESCRIPTION
**Pull request by kwalrath**

### CODE REVIEW

File: src/tools/dartdevc/faq.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ For more information, see [SDK issue #27776.][27776]
 ### Where can I see known problems with dartdevc?
 
 Issues are in the Dart SDK repo
-with the label [area-dev-compiler.](https://github.com/dart-lang/sdk/issues?q=is%3Aissue%20is%3Aopen%20label%3Aarea-dev-compiler)
+with the label [web-dev-compiler.](https://github.com/dart-lang/sdk/issues?q=is%3Aissue%20is%3Aopen%20label%3Aweb-dev-compiler)
```
Comment (vsmenon) [inline src/tools/dartdevc/faq.md]: Can this say:

labels [area-web] and [web-dev-compiler]

?  Or perhaps just [area-web]?  Our team will assign sub-areas if they aren't there.

Comment (kwalrath) [reply src/tools/dartdevc/faq.md]: Sure, I can add [area-web]. I feel like it's helpful to include [web-dev-compiler], so they don't see _too_ many issues (128 vs. 966 right now!).

Comment (kwalrath) [reply src/tools/dartdevc/faq.md]: I thought all dartdevc bugs would have the label area-web, but some of them have [other areas](https://github.com/dart-lang/sdk/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Aweb-dev-compiler+-label%3Aarea-web). So... do we really want to include the area-web label?

Comment (kwalrath) [reply src/tools/dartdevc/faq.md]: For context, here's where the text is:

https://dart.dev/tools/dartdevc/faq#where-can-i-see-known-problems-with-dartdevc

Comment (vsmenon) [reply src/tools/dartdevc/faq.md]: Yes, I'll be moving those issues to `area-web` (thanks for the heads up :-)).  We'll be using that for triaging.

**Comment by kwalrath** [comment]
ptal

**Pull request merged**